File: simply linked list/python/linkedList.py
class Node:
    # function to initialize the node object
    def __init__(self, data):
        self.data = data     # assign data
        self.next = None     # Initialize next as null



class Linkedlist:
    # Function to initialize the liked list object
    def __init__(self):
        self.head = None



if __name__ == '__main__':
    # start with the empty list
    llist = Linkedlist()

    llist.head = Node(1)
    second = Node(2)
    third = Node(3)

    llist.head.next = second     # link first node with second 

    second.next = third          # link second node with third

File: simply linked list/cpp/linkedList.cpp
#include <bits/stdc++.h>
using namespace std;

class Node{
    public:
        int data;
        Node* next;
};

// Program to create a simple linked list with 3 nodes

int main(){
    Node* head = NULL;
    Node* second = NULL;
    Node* third = NULL;

    // allocate 3 nodes in the heap
    head = new Node();
    second = new Node();
    third = new Node();

    // Assign data in first node 
    head->data = 1;         
    head->next = second;

    
    // assign data to second node
    second->data = 2;
    second->next = third;

    // assign data to third node
    third->data = 3;
    third->next = NULL;   /* next pointer 
    of the third block is made NULL to indicate 
    that the linked list is terminated here */

    return 0;
}